content/pl/_index.html

```diff
@@ -45,12 +45,12 @@ <h2>The Challenges of Migrating 150+ Microservices to Kubernetes</h2>
         <br>
         <br>
         <br>
-        <a href="https://events.linuxfoundation.org/events/kubecon-cloudnativecon-europe-2020/" button id="desktopKCButton">Weź udział w KubeCon w Amsterdamie 30.03-2.04.2020</a>
+        <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/?utm_source=kubernetes.io&utm_medium=nav&utm_campaign=kccnceu20" button id="desktopKCButton">Weź udział w KubeCon w Amsterdamie (lipiec/sierpień)</a>
```

Comment:
do we really want to include parameters for tracking the source like utm_source, utm_medium and so on?

Reply:
Good question! But again, this is for upstream maintainers.